### ISSUE TITLE
Cifar-10

### ISSUE THREAD

**Issue by seriousmac** [LeeYK/Deep-ing-Source#13]
crlinux@crlinux-All-Series:~/CR/Deep-ing-Source/CR-03$ python cifar10_train.py
>> Downloading cifar-10-binary.tar.gz 100.0%
Successfully downloaded cifar-10-binary.tar.gz 170052171 bytes.
Filling queue with 20000 CIFAR images before starting to train. This will take a few minutes.
2017-08-09 18:09:28.373064: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
2017-08-09 18:09:28.373082: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-08-09 18:09:28.373086: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2017-08-09 18:09:28.373090: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-08-09 18:09:28.373093: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
2017-08-09 18:09:28.502313: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:893] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2017-08-09 18:09:28.502600: I tensorflow/core/common_runtime/gpu/gpu_device.cc:940] Found device 0 with properties: 
name: GeForce GTX 1080
major: 6 minor: 1 memoryClockRate (GHz) 1.8475
pciBusID 0000:01:00.0
Total memory: 7.92GiB
Free memory: 7.41GiB
2017-08-09 18:09:28.502613: I tensorflow/core/common_runtime/gpu/gpu_device.cc:961] DMA: 0 
2017-08-09 18:09:28.502617: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971] 0:   Y 
2017-08-09 18:09:28.502623: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1030] Creating TensorFlow device (/gpu:0) -> (device: 0, name: GeForce GTX 1080, pci bus id: 0000:01:00.0)
2017-08-09 18:09:31.147946: E tensorflow/stream_executor/cuda/cuda_dnn.cc:352] Loaded runtime CuDNN library: 6021 (compatibility version 6000) but source was compiled with 5110 (compatibility version 5100).  If using a binary install, upgrade your CuDNN library to match.  If building from sources, make sure the library loaded at runtime matches a compatible version specified during compile configuration.
2017-08-09 18:09:31.148111: F tensorflow/core/kernels/conv_ops.cc:671] Check failed: stream->parent()->GetConvolveAlgorithms(&algorithms) 
Aborted (core dumped)

**Comment by seriousmac** [comment]
Download has completed.

1. Solution for warning
->
import os
os.environ['TF_CPP_MIN_LOG_LEVEL']='2'
import tensorflow as tf

or

Try export TF_CPP_MIN_LOG_LEVEL=2

2. Error
Loaded runtime CuDNN library: 6021 (compatibility version 6000) but source was compiled with 5110 (compatibility version 5100)
---------------------------------------------------------------------------------------------------------------------------------
To solve this problem, install CuDNN 5.1(5110) (instead of CuDNN 6.0/6021)

3. maybe because of 3

**Comment by seriousmac** [comment]
Result:
2017-08-10 21:28:24.278282: step 999990, loss = 0.10 (7177.0 examples/sec; 0.018 sec/batch)
